packages/schema/src/config/webpack.ts

```diff
@@ -151,8 +151,12 @@ export default defineUntypedSchema({
       fontUrl: { esModule: false, limit: 1000 },
       imgUrl: { esModule: false, limit: 1000 },
       pugPlain: {},
+
+      /**
+       * See [vue-loader](https://github.com/vuejs/vue-loader) for available options.
+       * @type {Partial<typeof import('vue-loader')['VueLoaderOptions']>}
+       */
       vue: {
-        productionMode: { $resolve: async (val, get) => val ?? !(await get('dev')) },
```

Comment:
This seemed like an [old option](https://github.com/vuejs/vue-loader/pull/1208) that is no longer in the `vue-loader` plugin's codebase.